specifications/catalog/catalog.protocol.md

```diff
@@ -56,17 +54,31 @@ This section describes how the IDS Information Model maps to DCAT resources.
 
 An `Asset Entry` is a [DCAT Dataset](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset) with the following attributes:
 
-#### 3.1.1 dcat:haPolicy
+#### 3.1.1 odrl:hasPolicy
 
+a)
 An asset entry Dataset may have 0..N `hasPolicy` attributes that contain an ODRL `Offer` defining the usage control policy associated with the asset. Offers must NOT contain any
 target attributes. The target of an offer is the asset associated with the containing asset entry. If an entry is has no associated policies (hasPolicy does not contain any offer
 entries), the asset is does not have any usage control policy.
 
+b)
+An asset entry Dataset shall not contain an ODRL `Offer`. Offers can only appear at `Distributions`. 
+
+> Note: As `odrl:hasPolicy rdfs:domain odrl:Asset` and `AssetEntry isA dcat:Dataset`
+
 ### 3.2 Distributions
 
 An asset may contain 0..N [DCAT Distributions](https://www.w3.org/TR/vocab-dcat-3/#Class:Distribution). Each distribution must have at least one `DataService` which specifies where
 the distribution is obtained. Specifically, a `DataService` specifies the endpoint for initiating a `ContractNegotiation` and `AssetTransfer`.
 
+A) 
+An Distribution must not have any `hasPolicy` attribute.
```

Comment:
I thought the minimum agreed upon policy is "USE" without constraints. When did we start to allow datasets/distributions without any contract offer/policy?

Reply:
Didn't want to express that, only thought about alternative patterns: 
a) at the Dataset-level only, or
b) at the Distribution-level only.
But this has become obsolete by our discussion this morning anyway.